elasticms-cli/src/Client/MediaLibrary/MediaLibrarySync.php

```diff
@@ -253,4 +273,36 @@ private function getMetadata(string $path): array
 
         return [];
     }
+
+    private function getFolderPath(): string
+    {
+        if (!$this->options->hashFolder) {
+            return $this->options->folder;
+        }
+
+        $folderZip = $this->getFileByHash($this->options->folder);
+        $zip = new \ZipArchive();
+
+        if (true !== $open = $zip->open($folderZip)) {
+            throw new \RuntimeException(\sprintf('Failed opening zip %s (ZipArchive %s)', $folderZip, $open));
+        }
+
+        $filesPath = $folderZip.'_unzipped';
```

Comment:
using tempnam? to avoid conflicts?

Reply:
https://github.com/ems-project/elasticms/pull/555/commits/0b54089ba4448e28847994215f7ccb7dc8e262e4